physionet-django/console/views.py

```diff
@@ -274,6 +275,17 @@ def submission_info(request, project_slug):
         LOGGER.info("The editor for the project {0} has been reassigned from "
                     "{1} to {2}".format(project, user,
                                         reassign_editor_form.cleaned_data['editor']))
+    elif 'embargo_files' in request.POST:
+        embargo_files_days_form = forms.EmbargoFilesDaysForm(data=request.POST)
+        if settings.SYSTEM_MAINTENANCE_NO_UPLOAD:
+            raise ServiceUnavailable()
+        elif embargo_files_days_form.is_valid():
+            embargo_days_from_form = embargo_files_days_form.cleaned_data['embargo_files_days']
+            if embargo_days_from_form == 0:
```

Comment:
instead of `embargo_days_from_form`, just `days` might be more readable? it's clear from the context that we are talking about an embargo.

i'd say the same for `embargo_files_days_form` too i think. something like `embargo_form` seems nicer.

Reply:
I agree for the most part but we were trying to keep this as specific as possible to the files being under embargo. Therefore I keep using the word files in most places. As I think you mentioned, we may want to embargo additional content in the future. 

Perhaps: 
`embargo_days_from_form` -> `files_days`
`embargo_files_days_form` -> `embargo_files_form` or `files_form` (though this could easily be used elsewhere)

Reply:
I know the post is `embargo_files`, so I don't need to be told in every subsequent variable that we are talking about embargo of the files. I just need to know that the variable contains `days` etc. 

Adding the extra information to the variable name interferes with readability for me, but I'm okay with whatever you end up with after giving it some thought. 

It's true that I think we should keep in mind that we may extend the embargo functionality to allow descriptions to be embargoed too, but (in most places) I don't think general language is counter to this. 

(e.g. an "embargo form" for me seems fine, because my guess is we would end up extending this form in the event we do allow embargoed descriptions).

Reply:
@tompollard I've changed to the variable names you suggested. If we add the ability to embargo the project description we can always revisit this if necessary.